compiler/src/dmd/identifier.d

```diff
@@ -300,6 +300,33 @@ nothrow:
         sv.value = id;
     }
 
+    extern (D) static Identifier idPoolNormalize(scope const(char)[] s, bool isNormalized)
```

Comment:
No documentation as to what this function does.

Reply:
For this PR it is the same as all the other ``idPool``'s.

In next PR, it is meant to handle normalization and string interning.

But yes, I can add comments.

Reply:
Documented.